rst_files/getting_started.rst

```diff
@@ -336,38 +303,6 @@ which provides a static HTML representations of notebooks
 
 QuantEcon also hosts the `QuantEcon Notes <http://notes.quantecon.org/>`_ website, where you can upload and share your notebooks with other economists and the QuantEcon community
 
-.. _julia_repl: 
-
-The REPL
-------------
-
-While we have not emphasized it, on any :ref:`JupyterHub <jl_jupyterhub>` or local Jupyter 
-installation you will also have access to the Julia REPL
-
-This is a Julia specific terminal disconnected from the graphical interface of 
-Jupyter, and becomes increasingly important as you learn Julia
-
-For example, the REPL is what we used in the beginning of this lecture to install 
-``InstantiateFromURL.jl`` and ``IJulia``
-
-To start the REPL in a typical Jupyter lab environment
-
-#. Choose "New Launcher"
-#. Choose a ``Julia 1.0`` Console
-
-Otherwise, if you have a local installation, then  
-
-#. Navigate to Julia through your menus or desktop icons (Windows, OSX), or
-
-#. Open a terminal and type ``julia`` (Linux)
-
-The REPL is one of the best places to add and remove packages, so a good test is to see the current status of the package manager
-
-.. code-block:: julia
-
```

Comment:
Again, this whole section is necessary for people to know to use julia.  It could all be tagged `Advanced`, but there is currently no other lecture it could go in.